pkg/build/controller/build/build_controller.go

```diff
@@ -1234,6 +1252,12 @@ func (bc *BuildController) createBuildPod(build *buildv1.Build) (*buildUpdate, e
 		if err != nil {
 			return nil, err
 		}
+		if bc.globalProxySet {
+			update, err = bc.createBuildProxyCAConfigMap(build, pod, update)
+			if err != nil {
+				return update, err
+			}
+		}
```

Comment:
@bparees per arch call my understanding is that we will always want to create this ConfigMap, and that ca bundle will contain all trust for the cluster.

For the sake of staging code these lines should be commented out until the new controller that injects the trusted CA bundle lands.

Reply:
OK so remove the "is proxy configured" qualifiers once @bparees confirms @adambkaplan 's understanding from the arch call

Reply:
commenting out part of next push

Reply:
you should always create the TrustedCA configmap, it should be annotated for injection, and you should always mount it into your pod.  In some cases this will mean that you mount an empty/non-existent trust bundle into your pod.  This is considered desirable behavior.

and yes, until the trusted CA controller exists, you do not wan to actually mount CM.

Reply:
@bparees we should clarify - we want to explicitly reference the injected key so that the build pod doesn't start until the trusted CA controller adds the ca-bundle, as we do with the service-ca bundle for the registry.

Reply:
sorry, for build-pods you're correct.  they are a bit of a special case.

Reply:
see https://github.com/openshift/openshift-controller-manager/pull/12#discussion_r309415769 I plan to get some code along those lines in the commented out section to help capture this